src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts

```diff
@@ -0,0 +1,320 @@
+
+/* eslint-disable max-len */
+import cuid from 'cuid'
+import { Rule, Result, Engine } from '@cloudgraph/sdk'
+import 'jest'
+
+import Azure_NIST_800_53_61 from '../rules/azure-nist-800-53-rev4-6.1'
+import Azure_NIST_800_53_62 from '../rules/azure-nist-800-53-rev4-6.2'
+import Azure_NIST_800_53_63 from '../rules/azure-nist-800-53-rev4-6.3'
+import Azure_NIST_800_53_64 from '../rules/azure-nist-800-53-rev4-6.4'
+import { initRuleEngine } from '../../../utils/test'
+
+export interface SiteConfig {
```

Comment:
are these exported types needed?

Reply:
No, I will remove everything that is not used